rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -128,13 +128,12 @@
     },
     "harvested": {
       "title": "Harvested",
-      "description": "Document is harvested or not, will disable record edition or similar.",
+      "description": "Whether the document is harvested or not, will disable record edition or similar.",
```

Comment:
or similar ___?

Reply:
Indeed, the source could be clearer. But I'm wondering if this description should exist in the first place. Our politic is to avoid description for fields that aren't displayed in the editor. I have to check.